sdk/chaos/Azure.ResourceManager.Chaos/src/autorest.md

```diff
@@ -7,8 +7,8 @@ azure-arm: true
 csharp: true
 library-name: Chaos
 namespace: Azure.ResourceManager.Chaos
-#tag: package-2023-11
-require: https://github.com/Azure/azure-rest-api-specs/blob/4af52aaac2c3b4af4a0e61378d33c5bc050e65e2/specification/chaos/resource-manager/readme.md
+tag: package-2024-01
```

Comment:
The specified tag is same as the default tag in readme.md, should be removed or commented out.

Reply:
Good catch, pushed up change now!

Reply:
Thank you @live1206